utilities/src/main/java/io/syndesis/qe/Component.java

```diff
@@ -69,9 +69,11 @@
      * @return list of pods of syndesis components
      */
     public static List<Pod> getComponentPods() {
-        return OpenShiftUtils.getInstance().pods().withLabel("syndesis.io/component").list().getItems().stream()
+        List<Pod> syndesisComponents = OpenShiftUtils.getInstance().pods().withLabel("syndesis.io/component").list().getItems().stream()
             .filter(p -> !StringUtils.endsWithAny(p.getMetadata().getName(), new String[] {"build", "deploy"}))
             .filter(p -> !"integration".equals(p.getMetadata().getLabels().get("syndesis.io/component")))
             .collect(Collectors.toList());
+        syndesisComponents.addAll(OpenShiftUtils.getInstance().pods().withLabel("app", "jaeger").list().getItems()); //jaeger operator
```

Comment:
I wouldn't count jaeger-operator as syndesis component, rather the jaeger instance that is created by the operator

Reply:
Jaeger instance syndesis-jaeger is already counted by the first search because it contains withLabel("syndesis.io/component") . However, I have added the Jaeger operator (which is created by Syndesis) here because getAllComponents() contains the Jaeger operator too and the comparison in the Syndesis#isReady https://github.com/syndesisio/syndesis-qe/blob/master/utilities/src/main/java/io/syndesis/qe/resource/impl/Syndesis.java#L106 was not equal.

Reply:
But instead of update `Component.getComponentPods()`, I can edit `Syndesis.isReady()`. Is that comparison necessary ? `syndesisPods.size() == components.size()`